web/src/oEmbed.js

```diff
@@ -45,17 +45,18 @@ async function getClaim(requestUrl) {
 // Generate
 // ****************************************************************************
 
-function generateOEmbedData(claim, referrerQuery) {
+function generateOEmbedData(claim, embedlyReferrer, timestamp, referral) {
   const { value, signing_channel: authorClaim } = claim;
 
   const claimTitle = value.title;
   const authorName = authorClaim ? authorClaim.value.title || authorClaim.name : 'Anonymous';
   const authorUrlPath = authorClaim && authorClaim.canonical_url.replace('lbry://', '').replace('#', ':');
   const authorUrl = authorClaim ? `${URL}/${authorUrlPath}` : null;
   const thumbnailUrl = value && value.thumbnail && value.thumbnail.url && getThumbnailCdnUrl(value.thumbnail.url);
+
+  const embedUrl = generateEmbedUrlEncoded(claim.name, claim.claim_id, timestamp, referral);
   const videoUrl =
-    generateEmbedUrlEncoded(claim.name, claim.claim_id) +
-    (referrerQuery ? `r=${encodeURIComponent(escapeHtmlProperty(referrerQuery))}` : '');
+    embedUrl + (embedlyReferrer ? `referrer=${encodeURIComponent(escapeHtmlProperty(embedlyReferrer))}` : '');
```

Comment:
Just to be safe, we should probably use the escapeHtmlProperty function on the referral and timestamp parameters also.

Reply:
Or it can be escaped out in the generateEmbedUrl function too?